checks/network.py

```diff
@@ -57,3 +57,19 @@ def checkNICSpeed(lines):
                     if 'GbE' in nic or 'Gigabit' in nic:
                         return [LEVEL_WARNING, "Slow Network Connection", "Your gigabit-capable network card is only connecting at 100mbps. This may indicate a bad network cable or outdated router / switch which could be impacting network performance."]
     return None
+
+
+x264stream_re = re.compile(r"stream")
+
+
+def checkDynamicBr(lines):
```

Comment:
To make the function a little more obvious when scrolling through, please rename it to `checkDynamicBitrate`.